docs/developers/README.md

```diff
@@ -202,7 +202,6 @@ The following directories are covered by the
 relied on for stability:
 
 - `/tekton/results`
```

Comment:
My belief is none of the internal paths are stable and should avoid consuming them. But `results` has been around since past many releases and might have been integrated by many consumers already.